importer/importer_utils.py

```diff
@@ -439,3 +439,13 @@ def create_wlm_url(country, language, id):
     url_base = "https://tools.wmflabs.org/heritage/api/api.php?action=search&format=json&srcountry={}&srlanguage={}&srid={}"
     return url_base.format(
         country, language, quote(id))
+
+
+def create_site_instance(language, family):
+    """
+    Create an instance of a Wiki site.
```

Comment:
I would shorten this down to "Convenience function to create an instance of a Wiki site." or possibly "Create an instance of a Wiki site (convenience function)."